src/interaction/interaction_elph.h

```diff
@@ -27,50 +32,52 @@
  * 3) for polar materials, adds the long-range Frohlich interaction.
  */
 // TODO: add flag to let user decide whether to use or not polar corrections
-class InteractionElPhWan {
+class InteractionElPhWan: public InteractionElPhBase {
+
+private:
 
-  Crystal crystal;
-  PhononH0 phononH0;
-  int numPhBands, numElBands, numWsR1Vectors, numWsR2Vectors;
-  std::vector<Eigen::Tensor<double, 3>> cacheCoupling;
-  bool usePolarCorrection = false;
-  const int phaseConvention;
```

Comment:
phase convention should go back into base class -- this is to account for the fact that "giustino" convention uses Re, Rp, and JDFTx uses Re, Re'. This changed the convention of the k and q -> k' math. 

Therefore, we need this phaseConvention in the base class